### PR TITLE
Logs UI Improvement

### PR DESCRIPTION
**Pull request by FabulousCupcake**
Updates Logs page to look slightly better and fully utilize semantic-ui's `<Label>` features.

<details>
<summary>Before</summary>
Screen Shot 2021-11-07 at 23 08 04
</details>

<details>
<summary>After</summary>
Screen Shot 2021-11-07 at 23 07 41
</details>

### CODE REVIEW

File: app/pages/Logs.js [hunk 1]
```diff
@@ -7,6 +7,23 @@ const remote = require('@electron/remote');
...
+  info: 'blue',
+  warning: 'yellow',
+  error: 'red',
+  debug: 'darkgrey',
+};
+const STATUS_ICON_MAP = {
+  success: 'check',
+  info: 'info circle',
+  warning: 'warning sign',
+  error: 'window close',
```
Comment (FabulousCupcake) [inline app/pages/Logs.js:21]:
```suggestion
  error: 'x',
```
Should actually be `x`; `window close`, `close`, and `delete` all makes the button clickable with weird margins.

File: app/pages/Logs.js [hunk 2]
```diff
@@ -27,24 +44,7 @@ class Logs extends React.Component {
...
-      case 'error':
-        return 'red';
-      case 'debug':
-        return 'black';
-      default:
-        return 'grey';
-    }
-  }
-
+
```
Comment (FabulousCupcake) [inline app/pages/Logs.js:47]:
```suggestion
```

**Comment by Xzandro** [comment]
Thanks for the PR. This looks good to me. Could you maybe provide examples for warnings and errors as well?

**Comment by FabulousCupcake** [comment]
Screen Shot 2021-11-08 at 00 44 08

Here's some error I have atm; no warnings but it should be ok

File: app/pages/Logs.js [hunk 3]
```diff
@@ -53,17 +52,18 @@ class Logs extends React.Component {
             <Feed.Event>
               <Feed.Content>
                 <Feed.Summary>
-                  <Label size="mini" color={this.labelColor(entry.type)}>
-                    {capitalize(entry.type)}
+                  <Label color={determineLabelColor(entry.type)} image horizontal>
+                    <Icon name={determineLabelIcon(entry.type)} />
```
Comment (Xzandro) [inline app/pages/Logs.js:56]: You do use Icon here, but did not define it at the top.

**Review by Xzandro — CHANGES_REQUESTED** [review]

File: app/pages/Logs.js (hunk 3 above)
Comment (FabulousCupcake) [reply app/pages/Logs.js:56]: Whoops, should be fixed with d538230

**Comment by Xzandro** [comment]
I personally liked it more with the divider added between the log entries. Currently it looks a bit noisy without it present. Would you mind adding that back? Otherwise this looks fine.

**Comment by FabulousCupcake** [comment]
IMO it's perfectly readable now that the colored label is longer (it helps to discern visually the separation between feed items), but maybe that's just me.

File: app/pages/Logs.js [hunk 4]
```diff
@@ -53,17 +52,18 @@ class Logs extends React.Component {
...
+                    {entry.name && <Label.Detail>{entry.name}</Label.Detail>}
                   </Label>
-                  {capitalize(entry.source)} {entry.name ? ` - ${entry.name}` : ''} <Feed.Date>{entry.date}</Feed.Date>
+                  <Feed.Date>{entry.date}</Feed.Date>
                 </Feed.Summary>
                 <Feed.Extra>
                   <div dangerouslySetInnerHTML={{ __html: entry.message }} />
                 </Feed.Extra>
               </Feed.Content>
             </Feed.Event>
```
Comment (FabulousCupcake) [inline app/pages/Logs.js:66]:
```suggestion
            </Feed.Event>
          <Divider />
```

Comment (FabulousCupcake) [reply app/pages/Logs.js:66]: @Xzandro you can commit this and merge if you'd like to keep it.

Comment (Xzandro) [reply app/pages/Logs.js:66]: Thanks for adding that back. I will leave it in. As you said, these things are a little bit subjective, but it's just more friendly looking in my opinion.

File: app/pages/Logs.js [hunk 5]
```diff
@@ -53,17 +52,19 @@ class Logs extends React.Component {
...
-            <Divider />
+          <Divider />
```
Comment (FabulousCupcake) [inline app/pages/Logs.js:67]:
```suggestion
            <Divider />
```

Comment (FabulousCupcake) [reply app/pages/Logs.js:67]: 😔

**Pull request merged**